connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerJDBCIntegrationSuite.scala

```diff
@@ -156,7 +156,7 @@ abstract class DockerJDBCIntegrationSuite
         .newHostConfig()
         .withNetworkMode("bridge")
         .withPrivileged(db.privileged)
-        .withPortBindings(PortBinding.parse(s"$dockerIp:$externalPort:${db.jdbcPort}"))
+        .withPortBindings(PortBinding.parse(s"$externalPort:${db.jdbcPort}"))
```

Comment:
Three parts binding seems not to work on Colima, removing IP part works for both